core/templates/dev/head/pages/exploration_editor/ParameterMetadataService.js

```diff
@@ -19,9 +19,11 @@
 oppia.factory('ParameterMetadataService', [
   'ExplorationStatesService', 'ExpressionInterpolationService',
   'ExplorationParamChangesService', 'GraphDataService',
+  'ParamMetadataObjectFactory',
   function(
       ExplorationStatesService, ExpressionInterpolationService,
-      ExplorationParamChangesService, GraphDataService) {
+      ExplorationParamChangesService, GraphDataService,
+      ParamMetadataObjectFactory) {
     var PARAM_ACTION_GET = 'get';
```

Comment:
@seanlip, This value is also used in ParamMetadataObjectFactory but without any variable, I think maybe we can make this as an oppia.constant to use this in both places or is it okay to leave it as-is?

Reply:
@hoangviet1993, can you create constants for 'get' and 'set' as these are also being used in ParamMetadataObjectFactory?

Reply:
Done!